### DISCUSSION TITLE
Get all error messages from validation

### DISCUSSION THREAD

**Discussion by wit3** [spatie/laravel-data#435] [Q&A] [answered]
It's possible to retrive all fields that fail validation instead of one single message that report

`first error and (and 1 more)`

like when the validation its in a request.

but using 
```php
DataClass::validate($payload);
```

**Comment by venux92** [comment, marked as answer]
You can get the errors from the `ValidationException`

```php
if ($e instanceof ValidationException) {
   return $e->validator->errors();
}
```
